telegram-bot/notes.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	tbot "github.com/go-telegram-bot-api/telegram-bot-api"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type notesData struct {
+	Name    string
+	Content string
+}
+
+func savenote(ID int64, msgtext string, client mongo.Client) {
+
+	collection := client.Database("test").Collection("SavedNote")
+
+	args := strings.Fields(msgtext)
+
+	var check notesData
+	_ = collection.FindOne(context.TODO(), bson.M{"name": args[1]}).Decode(&check)
+	if (notesData{}) == check {
+		if len(args) >= 3 {
+			data := notesData{
+				Name:    args[1],
+				Content: strings.Join(args[2:], " "),
```

Comment:
I guess `Link` would be a better name since we would be saving the Article Link and its Title for the most part ? Kinda like Hackernews?

Reply:
If it will only save the link, I will modify it to save the link with inline button( like I have done for the general set of links in the bot).

Reply:
But if I do that, we will have to parse the link smh . It can either the last arguement. Or we can  pass it in markdown syntax and parse it later!

What do you think?

Reply:
Did you mean changing `Content` to link? That won't be the right way imo. We can have three fields i.e 
* Name ( It is the command with which bot fetches the note )
* Title/Description 
* Link

Reply:
Sounds fine 👍